### DISCUSSION TITLE
Problem with integration sdk embed-> Unexpected token '<', "<!doctype "... is not valid JSON

### DISCUSSION THREAD

**Discussion by falehenrique** [Web3Auth/Web3Auth#577] [Help]
Hello!!

I'm receive this error on integration with torus embed sdk:

`
_app-ba06549dde59026c.js:1 Unrecognized feature: 'class="torusIframe"'.
(anonymous) @ _app-ba06549dde59026c.js:1
x @ _app-ba06549dde59026c.js:1
await in x (async)
init @ _app-ba06549dde59026c.js:1
await in init (async)
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ main-23decb939cc9dedb.js:1
(anonymous) @ main-23decb939cc9dedb.js:1
n @ _app-ba06549dde59026c.js:1
s @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ main-23decb939cc9dedb.js:1
(anonymous) @ main-23decb939cc9dedb.js:1
n @ _app-ba06549dde59026c.js:1
s @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
(anonymous) @ _app-ba06549dde59026c.js:1
onClick @ _app-ba06549dde59026c.js:1
Oe @ framework-0ba0ddd33199226d.js:1
Ae @ framework-0ba0ddd33199226d.js:1
(anonymous) @ framework-0ba0ddd33199226d.js:1
Ir @ framework-0ba0ddd33199226d.js:1
Ur @ framework-0ba0ddd33199226d.js:1
(anonymous) @ framework-0ba0ddd33199226d.js:1
ss @ framework-0ba0ddd33199226d.js:1
Le @ framework-0ba0ddd33199226d.js:1
Hr @ framework-0ba0ddd33199226d.js:1
Kn @ framework-0ba0ddd33199226d.js:1
Qn @ framework-0ba0ddd33199226d.js:1
DevTools failed to load source map: Could not parse content for https://app.tor.us/v1.35.5/js/chunk-vendors.d69387eb.js.map: Unexpected token '<', "<!doctype "... is not valid JSON
app.3c6f50ff.js:1 VUE_APP_TORUS_BUILD_ENV production
chunk-vendors.d69387eb.js:1 SyntaxError: "undefined" is not valid JSON
    at JSON.parse (<anonymous>)
    at OpenLoginStore.set (chunk-vendors.d69387eb.js:1:520273)
    at OpenLoginHandler.getActiveSession (app.3c6f50ff.js:1:15019)
    at Store.rehydrate (app.3c6f50ff.js:1:196114)
    at Array.wrappedActionHandler (chunk-vendors.d69387eb.js:80:7281)
    at Store.dispatch (chunk-vendors.d69387eb.js:80:8777)
    at Store.boundDispatch [as dispatch] (chunk-vendors.d69387eb.js:80:4370)
    at VueComponent.mappedAction (chunk-vendors.d69387eb.js:80:11855)
    at VueComponent.created (app.3c6f50ff.js:1:35825)
    at invokeWithErrorHandling (chunk-vendors.d69387eb.js:74:23583)
    at callHook$1 (chunk-vendors.d69387eb.js:74:32581)
    at s._init (chunk-vendors.d69387eb.js:74:45320)
    at new VueComponent (chunk-vendors.d69387eb.js:74:46768)
    at createComponentInstanceForVnode (chunk-vendors.d69387eb.js:74:37895)
    at init (chunk-vendors.d69387eb.js:74:36390)
    at createComponent (chunk-vendors.d69387eb.js:74:57315)
    at createElm (chunk-vendors.d69387eb.js:74:56836)
    at Vue.patch [as __patch__] (chunk-vendors.d69387eb.js:74:62706)
    at s._update (chunk-vendors.d69387eb.js:74:29938)
    at Vue.p (chunk-vendors.d69387eb.js:74:30715)
    at Watcher.get (chunk-vendors.d69387eb.js:74:26552)
    at new Watcher (chunk-vendors.d69387eb.js:74:26451)
    at mountComponent (chunk-vendors.d69387eb.js:74:30826)
    at Vue.$mount (chunk-vendors.d69387eb.js:74:83229)
    at 79718 (app.3c6f50ff.js:1:240678)
    at __webpack_require__ (app.3c6f50ff.js:1:490840)
    at app.3c6f50ff.js:1:499156
    at __webpack_require__.O (app.3c6f50ff.js:1:491310)
    at app.3c6f50ff.js:1:499208
    at app.3c6f50ff.js:1:499214 'Failed to rehydrate'
h @ chunk-vendors.d69387eb.js:1
rehydrate @ app.3c6f50ff.js:1
await in rehydrate (async)
wrappedActionHandler @ chunk-vendors.d69387eb.js:80
dispatch @ chunk-vendors.d69387eb.js:80
boundDispatch @ chunk-vendors.d69387eb.js:80
mappedAction @ chunk-vendors.d69387eb.js:80
created @ app.3c6f50ff.js:1
invokeWithErrorHandling @ chunk-vendors.d69387eb.js:74
callHook$1 @ chunk-vendors.d69387eb.js:74
s._init @ chunk-vendors.d69387eb.js:74
VueComponent @ chunk-vendors.d69387eb.js:74
createComponentInstanceForVnode @ chunk-vendors.d69387eb.js:74
init @ chunk-vendors.d69387eb.js:74
createComponent @ chunk-vendors.d69387eb.js:74
createElm @ chunk-vendors.d69387eb.js:74
patch @ chunk-vendors.d69387eb.js:74
s._update @ chunk-vendors.d69387eb.js:74
p @ chunk-vendors.d69387eb.js:74
Watcher.get @ chunk-vendors.d69387eb.js:74
Watcher @ chunk-vendors.d69387eb.js:74
mountComponent @ chunk-vendors.d69387eb.js:74
Vue.$mount @ chunk-vendors.d69387eb.js:74
79718 @ app.3c6f50ff.js:1
__webpack_require__ @ app.3c6f50ff.js:1
(anonymous) @ app.3c6f50ff.js:1
__webpack_require__.O @ app.3c6f50ff.js:1
(anonymous) @ app.3c6f50ff.js:1
(anonymous) @ app.3c6f50ff.js:1
DevTools failed to load source map: Could not parse content for https://app.tor.us/v1.35.5/js/app.3c6f50ff.js.map: Unexpected token '<', "<!doctype "... is not valid JSON
`
image of the error:
https://drive.google.com/file/d/1QszkUf4takfhIYOiUAcoFp74x-Rk5O43/view?usp=sharing

I'm using this application:
https://lionfish-app-kwtf8.ondigitalocean.app/
or this:
https://shop.thecampariblockchain.io/

My package.json:
`  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "export": "npm run build && next export -o _static"
  },
 "@toruslabs/torus-embed": "1.35.5",`

The problem it's running with chrome, in brave is working and I was deployed the app in DigitalOcean. But I get the same error in localhost.

Here is the code:
`          
            let torusInstance = new Torus({});
            console.log("[DEBUG] the problem it's running here");
            await torusInstance.init({
              buildEnv: "production",
              enableLogging: true,
              network: {
                host: getNetworkName(config.networkType, config.blockchain),
              },
              showTorusButton: true,
              skipTKey: true,
            });
            const loginParams: TorusLoginParams = {};
            if (loginType !== undefined && loginType !== "passwordless") {
              loginParams.verifier = loginType;
            } 

            console.log("loginParams ==> ", loginParams);
            const [torusAddress] = await torusInstance.login(loginParams);
            console.log("[DEBUG] torusAddress", torusAddress);
            const torusProvider = torusInstance.provider;
            address = torusAddress;
            web3 = new Web3(torusProvider as AbstractProvider);
            ethereumProvider = torusProvider;`

Somebody can help me?

Tks..

**Comment by shahbaz17** [comment]
Hey @falehenrique 

Can you try updating `@toruslabs/torus-embed` to `v1.36.4`? And let us know if this error is still there?
